.travis.yml

```diff
@@ -22,6 +22,8 @@ matrix:
         - wget https://github.com/google/google-java-format/releases/download/google-java-format-1.7/google-java-format-1.7-all-deps.jar
       script:
         - find . -iname "*.java" | xargs java -jar google-java-format-1.7-all-deps.jar --dry-run
+        - git diff --name-only --diff-filter=AM -- '*.java'
+        - git diff --name-only --diff-filter=AM HEAD...$TRAVIS_BRANCH -- '*.java' | xargs java -jar google-java-format-1.7-all-deps.jar --dry-run --set-exit-if-changed
```

Comment:
Do we need all 3 of these commands now? or can we run just that last one?

Reply:
@seehamrun Sorry, I'm still testing this. Will take a look at this PR this weekend [Little busy at my work 😓]